lib/active_model/serializer.rb

```diff
@@ -233,6 +248,14 @@ def links
       self.class._links
     end
 
+    def meta
```

Comment:
Should we prefer  `_meta` even though there shouldn't be attribute method conflicts anymore?